lightning_pose/losses/losses.py

```diff
@@ -275,13 +273,11 @@ def __init__(
     ) -> None:
         super().__init__(data_module=data_module, log_weight=log_weight)
         self.loss_name = loss_name
-        self.error_metric = error_metric
 
         if loss_name == "pca_multiview":
             if mirrored_column_matches is None:
                 raise ValueError("must provide mirrored_column_matches in data config")
 
-        # TODO: solve issues with pca loss + data augmentation
         # the current data_module contains datasets that are loaded using augmentations. the
```

Comment:
is this comment updated?

Reply:
yes, this was from before our updates a few weeks ago when there was no data aug for unlabeled data